src/components/ListItem.jsx

```diff
@@ -1,11 +1,18 @@
 import './ListItem.css';
 import { useEffect, useState } from 'react';
-import { updateItem } from '../api/firebase';
+import { updateItem, deleteItem } from '../api/firebase';
 import { ONE_DAY_IN_MILLISECONDS } from '../utils/dates';
 
-export function ListItem({ name, listPath, itemId, dateLastPurchased }) {
+export function ListItem({
+	name,
+	listPath,
+	itemId,
+	dateLastPurchased,
+	daysUntilNextPurchase,
+}) {
 	const [isChecked, setIsChecked] = useState(false);
 	const [expired, setExpired] = useState();
+	const [purchaseUrgency, setPurchaseUrgency] = useState('');
```

Comment:
Are these being used anywhere in this file?

Reply:
Removed!